### PR TITLE
docs-fr | docs | reference | glossary | admission-controller.md

### PR DESCRIPTION
**Pull request by julienemo**

### CODE REVIEW

File: content/fr/docs/reference/glossary/admission-controller.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+aka:
+tags:
+- extension
+- security
+---
+Un morceau de code qui intercepte les requêtes adressées au serveur API de Kubernetes avant la persistance de l'objet.
+
+<!--more-->
+
+Les contrôleurs d'accès sont configurables pour le serveur API de Kubernetes. Un contrôleur d'accès peut être "validant"  ("validating"), "mutant" ("mutating"), ou
```
Comment (perriea) [inline content/fr/docs/reference/glossary/admission-controller.md]:
```suggestion
Les contrôleurs d'accès sont configurables pour le serveur API de Kubernetes. Un contrôleur d'accès peut être "validant"  ("validating"), "changant" ("mutating"), ou
```
Complex to find a translation to this word

Comment (julienemo) [reply content/fr/docs/reference/glossary/admission-controller.md]: I also considered "modifiant". How does that sound ? @perriea

Comment (perriea) [reply content/fr/docs/reference/glossary/admission-controller.md]: Perfect !

Comment (remyleone) [reply content/fr/docs/reference/glossary/admission-controller.md]: Modifiant c'est bien

File: content/fr/docs/reference/glossary/admission-controller.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+les deux. Tout contrôleur d'accès peut rejeter la demande. Les contrôleurs mutant peuvent modifier les objets qu'ils admettent ;
```
Comment (remyleone) [inline content/fr/docs/reference/glossary/admission-controller.md]: modifiant

File: content/fr/docs/reference/glossary/admission-controller.md (hunk 1 above)
Comment (julienemo) [reply content/fr/docs/reference/glossary/admission-controller.md]: @remyleone @perriea "Modifiant" it is then! Updated.

**Pull request merged**